src/material-experimental/mdc-slider/slider-thumb.scss

```diff
@@ -0,0 +1,4 @@
+.mat-mdc-slider .mat-ripple {
```

Comment:
shouldn't this go in slider.scss not slider-thumb.scss, since it targets the container class?

Reply:
@andrewseguin

Reply:
I think that the `.mat-mdc-slider` should be changed to the host's class (it doesn't have one now, but should be added, e.g.  `.mat-slider-visual-thumb`. Also, I didn't remember that the visual thumb's implementation is still in the `slider.ts`. I think it ought to be moved into its own file.

Reply:
Moving the `MatSliderVisualThumb` to a separate file causes circular deps

Reply:
D'oh yeah that makes sense. Unless you wanted to add some intermediate interface this is probably fine

Reply:
Yeah, I'm going to keep it as is. We tried using interfaces to move `MatSliderThumb` and `SliderAdapter` to separate files but it became very confusing very quickly so we ended up just collapsing everything to a single file